user_guide_src/source/changelogs/v4.2.7.rst

```diff
@@ -14,6 +14,7 @@ BREAKING
 
 - The default values of the parameters in :php:func:`set_cookie()` and :php:meth:`CodeIgniter\\HTTP\\Response::setCookie()` has been fixed. Now the default values of ``$secure`` and ``$httponly`` are ``null``, and these values will be replaced with the ``Config\Cookie`` values.
 -  ``Time::__toString()`` is now locale-independent. It returns database-compatible strings like '2022-09-07 12:00:00' in any locale.
+- The Rule of :php:func:`Validation\\Rule::required_without()` parameters are changed and this rule logic is fixed.
```

Comment:
```suggestion
- The Validation rule ``Validation\Rule::required_without()`` and ``Validation\StrictRules\Rule::required_without()`` parameters have been changed and the logic of these rule has also been fixed.
```

`:php:func:` is for creating a link to the function reference page. But it is not a function,
but a class method. So in this case, we need to use `:php:meth` instead.
But there is no page for the class reference.

Reply:
Ok, I will be careful next time.